include/video.h

```diff
@@ -188,18 +188,18 @@ enum class PixelFormat : uint8_t {
 	// FFmpeg Equivalent: AV_PIX_FMT_BGR24
 	BGR24_ByteArray = 24,
 
-	// 16.7M (32-bit) true colour; 8 bits per red/blue/green component;
-	// stored as packed uint32 data with highest 8 bits unused
+	// 16.7M (32-bit) true colour, 8 bits per red/blue/green component;
+	// stored as packed 32-bit data
 	//
-	// Stored as array of uint32_t in host native endianess.
-	// Each uint32_t is layed out as follows:
-	// (msb)8X 8R 8G 8B(lsb)
+	// Stored as array of uint8_t in BGRA memory order (endian agnostic)
```

Comment:
BGRX no? VGA cards don't have the concept of an alpha channel.

Reply:
Will rename to `Bgrx8888` across the board.